### DISCUSSION TITLE
Espanso is missing in Winget's repo

### DISCUSSION THREAD

**Discussion by PolpOnline** [espanso/espanso#1187] [Feature requests and Ideas]
In italian 🇮🇹
Ciao, ho notato che Espanso manca nella repository di Winget, un package manager creato da Microsoft per installare le applicazioni su Windows più velocemente.
La documentazione per creare i file manifest è disponibile [qua](https://docs.microsoft.com/en-us/windows/package-manager/package/manifest) e la repository dove aggiungere i manifest è [qua](https://github.com/microsoft/winget-pkgs).

In english 🇺🇸
Hi, I noticed that Espanso is missing in the repo of Winget, a package manager created by Microsoft to install applications on Windows faster.
The documentation to create the manifest files is available [here](https://docs.microsoft.com/en-us/windows/package-manager/package/manifest) and the repository where to add the manifest is [here](https://github.com/microsoft/winget-pkgs).

**Comment by federico-terzi** [comment]
Hey @blackcat-917,

Thank you for the suggestion! I'll make sure to add espanso to Winget in the future, even though I prefer to wait until it's out of preview-mode so that it has a stable specification.

So I guess we will have to wait for a while :)

Cheers :)

**Comment by PolpOnline** [comment]
Hi, WinGet is now at v1.0 (Meanwhile I switched to Linux but that's ok 😅). Should I make the manifest or you prefer to do it yourself?

**Comment by federico-terzi** [comment]
Hey @blackcat-917,

Sorry for not following up on this, it's been a pretty busy period! I'd say we should wait for #594 to be released, as the packaging format will change a bit, and there is no much point in supporting an outdated version. What do you think?

Cheers :)

**Comment by JeppeKlitgaard** [comment]
@federico-terzi,
I noticed you were doing some CI packaging stuff for `.deb` and MacOS brew - maybe this could be lumped in there with it?

It goes without saying that I am super excited for the next pre-release. It looks like there is some exciting stuff in there!

**Comment by federico-terzi** [comment]
Hey @JeppeKlitgaard,

Thanks for reminding me! I'll investigate

Cheers :)

**Comment by AucaCoyan** [comment]
#2142
